source/docs/hardware/hardware-basics/images/preemptive-troubleshooting/tighten-breaker.svg

```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 585.092 394.983"><defs><style>.cls-1{fill:#4c5657;}.cls-2{fill:#b1b4b0;}.cls-3{fill:#646964;}.cls-4{fill:#525655;}.cls-5{fill:#c82c3d;}.cls-6{fill:#0c100e;}.cls-7{fill:#9c9167;}.cls-8{fill:#83857d;}.cls-9{fill:#961f2b;}.cls-10{fill:#35393a;}.cls-11{fill:none;stroke:#0695cf;stroke-linecap:round;stroke-miterlimit:10;stroke-width:15px;}.cls-12{fill:#0695cf;}.cls-13{font-size:28.9078px;font-family:Verdana;}.cls-14{letter-spacing:-0.0098em;}.cls-15{letter-spacing:0em;}</style></defs><rect class="cls-1" x="130.4532" y="124.4281" width="323.4677" height="232.386" rx="28.0103"/><rect class="cls-2" x="66.9556" y="166.1753" width="115.4826" height="51.5432" rx="25.7716"/><rect class="cls-3" x="198.8521" y="125.1548" width="186.6699" height="230.0049" rx="14.2925"/><rect class="cls-4" x="268.7267" y="124.9564" width="53.514" height="230.0049" rx="14.2925"/><rect class="cls-5" x="284.1887" y="286.1463" width="22.59" height="39.9847" rx="10.4033"/><path class="cls-6" d="M267.82,141.8975l-45.9931,44.04a6.3606,6.3606,0,0,0,0,9.2711l5.7982,5.5519a7.063,7.063,0,0,0,9.6824,0l31.4267-30.092V141.17A6.816,6.816,0,0,0,267.82,141.8975Z"/><polygon class="cls-7" points="175.042 168.352 146.584 168.352 132.355 192.998 146.584 217.643 175.042 217.643 189.271 192.998 175.042 168.352"/><circle class="cls-8" cx="160.813" cy="192.9975" r="16.9404"/><rect class="cls-9" x="18.8116" y="158.2407" width="89.0393" height="67.5613" rx="8.5732"/><rect class="cls-2" x="402.6538" y="285.9052" width="115.4826" height="51.5432" rx="25.7716" transform="translate(920.7901 623.3535) rotate(-180)"/><polygon class="cls-7" points="410.05 335.272 438.508 335.272 452.737 310.626 438.508 285.981 410.05 285.981 395.821 310.626 410.05 335.272"/><circle class="cls-8" cx="424.279" cy="310.6262" r="16.9404"/><rect class="cls-9" x="477.241" y="277.8217" width="89.0393" height="67.5613" rx="8.5732" transform="translate(1043.5214 623.2047) rotate(180)"/><circle class="cls-10" cx="416.5675" cy="163.3289" r="19.1241"/><circle class="cls-10" cx="164.1767" cy="318.2399" r="20.8627"/><path class="cls-11" d="M216.3288,185.4678a56.0193,56.0193,0,1,1-25.1877-39.5826"/><polygon class="cls-12" points="168.294 157.699 212.561 171.5 202.385 126.262 168.294 157.699"/><path class="cls-11" d="M478.6253,297.0183a56.0192,56.0192,0,1,1-29.4-36.5625"/><polygon class="cls-12" points="427.821 274.718 473.339 283.551 458.236 239.713 427.821 274.718"/><text class="cls-13" transform="translate(94.448 35.7049)">Ensure nuts are tightened <tspan x="9.2876" y="34.6895">firmly and the brea</tspan><tspan class="cls-14" x="288.8662" y="34.6895">k</tspan><tspan class="cls-15" x="305.6914" y="34.6895">er is </tspan><tspan x="-8.6597" y="69.3789">attached to a rigid element.</tspan></text></svg>
```

Comment:
I think the text contained in this image should be in the document, that way it is translatable.

Reply:
Thanks. Addressed in 1f853b0.